### DISCUSSION TITLE
KeepassXC locking up after any change - Ubuntu 20.04.0 LTS

### DISCUSSION THREAD

**Discussion by Dinaden** [keepassxreboot/keepassxc#7986] [Q&A]
I guess since the upgrade to 2.7.1, though it feels longer, after making any change to the data, it's impossible to close the window. The only option seems to be to log off and on again. Any suggestion to try appreciated. I'm not a programmer, simply a user.

**Comment by droidmonkey** [comment]
Try disabling automatic saves in the settings.

**Reply by Dinaden** [reply]
Thanks for the suggestion. I'll be honest, I'm not sure what's cured it. I removed all the Save options and that made no difference. I used to always choose to manually save the database but after lots of fiddling, oh and reinstalling, I've simply left the first option' Automatically save after every change' and all's good. Cheers.
